libraries/chain/include/eosio/chain/webassembly/interface.hpp

```diff
@@ -16,28 +16,292 @@ namespace webassembly {
+          * The output buffer format should be valid input for set_parameters_packed.
+          * For each known parameter_id in the input sequence there should be an associated entry in the output sequence with the current encoded parameter_value.
+          *
+          * @brief Retrieve the blockchain config parameters.
+          * @ingroup privileged
+          *
+          * @param packed_parameter_ids - the input buffer with the format as described above.
+          * @param[out] packed_parameters - the output buffer with the format as described above.
+         */
          uint32_t get_parameters_packed( span<const char> packed_parameter_ids, span<char> packed_parameters) const;
+
+         /**
+          * Set the blockchain parameters.
+          * It allows a system contract the ability to set parameters in a flexible manner.
+          * The input buffer is a packed data stream which represents an encoded sequence of parameter_id:paramter_value pairs with the following format:
+          * |varuint32:sequence_length | varuint32:parameter_id | <various>:parameter_value | ...
+          * The encoding of parameter_values should be specific to the parameter being set.
+          * Having duplicate parameter_ids encoded in the sequence should result in aborting the transaction context.
+          * The presence of a parameter_id which is unknown OR which is known but tied to an unactivated consensus protocol
+          * should result in aborting the transaction context.
+          * There are no requirement for the ordering of items in the sequence.
+          *
+          * @brief Set the blockchain parameters in a flexible manner.
+          * @ingroup privileged
+          *
+          * @param packed_parameters - buffer to hold the packed data with the format described above.
+         */
          void set_parameters_packed( span<const char> packed_parameters );
-         uint32_t get_kv_parameters_packed(name, span<char>, uint32_t) const;
-         void set_kv_parameters_packed(name, span<const char>);
+
+         /**
+          * Gets the maximum key size, maximum value size, and maximum iterators of a kv database and returns the size of the data.
+          * The kv parameters are encoded as 16 bytes, representing four 32-bit little-endian values.
+          * +-------+---------------+---------------+---------------+---------------+
+          * | byte  | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 |10 |11 |12 |13 |14 |15 |
+          * +-------+---------------+---------------+---------------+---------------+
+          * | field |   version     |   key limit   |  value limit  | max iterators |
+          * +-------+---------------+---------------+---------------+---------------+
+          * | type  |      0        |   32-bits LE  |  32-bits LE   |  32-bits LE   |
+          * +-------+---------------+---------------+---------------+---------------+
```

Comment:
This table is not rendering correctly. Check https://www.doxygen.nl/manual/tables.html

Reply:
Thanks for the observation, but after trying the table solution the html table still was not rendering properly

/**
 * Gets the maximum key size, maximum value size, and maximum iterators of a kv database and returns the size of the data.
 * The kv parameters are encoded as 16 bytes, representing four 32-bit little-endian values.
 * <table>
 * <tr><th>byte <th> 0 | 1 | 2 | 3 <th> 4 | 5 | 6 | 7 <th> 8 | 9 | 10 | 11 <th> 12 | 13 | 14 | 15
 * <tr><td>field <td>version <td>key limit <td>value limit <td>max iterators
 * <tr><td>type <td>0 <td>32-bits LE <td>32-bits LE <td>32-bits LE
 * </table>
 *
 * @ingroup privileged
 * @param[out] packed_kv_parameters - the buffer containing the packed kv parameters.
 * @param max_version - has no effect, but should be 0.
 *
 * @return Returns the size required in the buffer (if the buffer is too small, nothing is written).
 *
*/

I switched to use codeblocks instead.